tests/gatekeeper/v1beta1-template/policy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
```

Comment:
You might consider an additional test with a `v1` template.  New users will likely use `v1`, but it will be a long time before the `v1beta1` template are gone.

Reply:
Our examples are tests too. I have changed all of our examples work with v1 template.